synapse/cortex.py

```diff
@@ -1031,6 +1032,11 @@ class Cortex(s_cell.Cell):  # type: ignore
             'description': 'Enable storm search interfaces for lookup mode.',
             'type': 'boolean',
         },
+        'storm:interface:scrape': {
+            'default': True,
+            'description': 'Enable built in scrape APIs when using the scrape interface.',
```

Comment:
Is this inverted?  This option should enable the scrape interface when using the built in APIs right?

Reply:
I don't think so. This is using the same logic as `storm:interface:search` which defaults to using the interface. The flag allows someone to disable it if they chose to do so.